lib/stages/visTranspose.cpp

```diff
@@ -344,3 +415,31 @@ void visTranspose::increment_chunk() {
     write_f = f_edge ? num_freq - f_ind : chunk_f;
     write_t = t_edge ? num_time - t_ind : chunk_t;
 }
+
+dset_id_t visTranspose::base_dset(dset_id_t ds_id) {
```

Comment:
This should really be implemented in the datasetManager. Make sure to leave an issue in case we merge it like this.